libs/coin-evm/src/adapters/ethers.ts

```diff
@@ -0,0 +1,39 @@
+import { ethers } from "ethers";
+import {
+  Transaction as EvmTransaction,
+  EvmTransactionEIP1559,
+  EvmTransactionLegacy,
+} from "../types";
```

Comment:
If you go along with this suggestion -> https://github.com/LedgerHQ/ledger-live/pull/3827/files/584f33d826f57b5efbd8797dbbf34ff86bf14b37#r1260880330

```suggestion
import { Transaction as EvmTransaction } from "../types";
```

Reply:
Done